pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js

```diff
@@ -36,6 +36,36 @@ const HAPPY = 'H'
 const TASK_URL = "/tasks/"
 const USER_URL = "/user-data/"
 const TODAY = new Date()
+
+const WEIGHTS = [
+    '15 paperclips',
+    '3 nickels',
+    '5 pieces of bubblegum',
+    '4 mullberries',
+    '2/3 a medium radish',
+    '461 sequins',
+    '25 miniature toadstools',
+    '5 acorns',
+    '1 small egg',
+    '231.5 grains of rice',
+    '5 spoonfuls of cinnamon',
```

Comment:
teaspoons? tablespoons? world's largest spoons? i don't know what my pet weighs!

Reply:
It's really 5.769 tsp of cinnamon but the pet's weight may fluctuate very slightly over the course of time.
I will make it 6 teaspoonfuls of cinnamon :D